src/components/DatePicker/index.ios.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import {Button, View} from 'react-native';
+import DatePicker from '@react-native-community/datetimepicker';
+import moment from 'moment';
+import ExpensiTextInput from '../ExpensiTextInput';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import Popover from '../Popover';
+import CONST from '../../CONST';
+import styles from '../../styles/styles';
+import colors from '../../styles/colors';
+import {propTypes, defaultProps} from './datepickerPropTypes';
+
+const iosDatepickerPropTypes = {
+    ...propTypes,
+    ...withLocalizePropTypes,
+};
+
+class DatepickerIOS extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isPickerVisible: false,
+            selectedDate: props.value ? moment(props.value).toDate() : null,
+        };
+
+        this.showPicker = this.showPicker.bind(this);
+        this.reset = this.reset.bind(this);
+        this.selectDate = this.selectDate.bind(this);
+        this.updateLocalDate = this.updateLocalDate.bind(this);
+    }
+
+    showPicker() {
```

Comment:
Do we need to `preventDefault` here? Is it necessary in the Android counterpart?

Reply:
I thinks yes, I'll check and update